### DISCUSSION TITLE
BW2 to BW25 Transition Page

### DISCUSSION THREAD

**Discussion by marc-vdm** [brightway-lca/brightway-documentation#131] [Documentation Request]
### Request Category

Changes to existing page

### Page Title

https://docs.brightway.dev/en/latest/content/faq/brightway.html

### Description

Could we have a page/notebook that explains for every `BW2` function to a `BW25` function that was changed? I'm assuming things like this already exist around, but it'd be great to have this in the FAQ as a 'how do I move my project to BW25?' entry

### Can you help?

- [ ] I could help with the content of this documentation request.
- [ ] I could help with the technical implementation of this documentation request.

**Comment by michaelweinold** [comment]
Hey, @marc-vdm! This makes sense, thank you for the suggestion.

> I'm assuming things like this already exist around (...)

At first glance (the usual repos), I couldn't find anything.

We could ask ChatGPT to write a short, self-contained script using perhaps:

 - [`difflib`](https://docs.python.org/3/library/difflib.html) (mentioned in [this Toward Data Science article](https://archive.ph/YbesF))
 - [project2vec](https://developers.redhat.com/articles/2021/10/06/find-and-compare-python-libraries-project2vec)
